pom.xml

```diff
@@ -428,15 +426,15 @@
             <dependency>
                 <groupId>com.github.spotbugs</groupId>
                 <artifactId>spotbugs-annotations</artifactId>
-                <version>${spotbugs.version}</version>
+                <version>${dep.plugin.spotbugs.version}</version>
```

Comment:
Looking at Maven Central, it looks like the plugin and annotations jars are only _sort of_ versioned in lockstep.

- [spotbugs-maven-plugin](https://search.maven.org/search?q=g:com.github.spotbugs%20AND%20a:spotbugs-maven-plugin&core=gav)
- [spotbugs-annotations](https://search.maven.org/search?q=g:com.github.spotbugs%20AND%20a:spotbugs-annotations&core=gav)

Not all plugin versions exist for the annotations jar, and vice versa. Additionally, the annotations jar tends to be released days before the plugin--sometimes weeks.

In this instance it's fine, but it may bite us in future basepom upgrades--something to watch out for.

Reply:
It does for 3.1.7, which irpnically is needed for Java 11. The core is out but not the maven plugin